### DISCUSSION TITLE
Ability to hide stacks not managed by Dockge

### DISCUSSION THREAD

**Discussion by TazTheManiac** [louislam/dockge#609] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

The ability to hide stacks not managed by Dockge from the side menu and from being counted as active, exited, or inactive.

### ✔️ Solution

Implement a setting under general where you could enable a switch to have Dockge hide and don't count stacks not managed by it. 

### ❓ Alternatives

Using custom CSS to hide the item from being displayed in the side list, but this don't help with the stack being counted as active, exited, or inactive.

**Comment by aparillo64** [comment]
The option to hide non-Dockge stacks would give a cleaner appearance and reduce the clutter on the interface.

**Comment by wey-gu** [comment]
Agree on such an option

**Comment by coyodi** [comment]
Thirded

**Comment by Brayyy** [comment]
I do appreciate that stacks that aren't managed by Dockge are dimmed and can't be modified, I do also think this would be a nice config option to clean up the UI. As an alternative, even being able to selectively hide non-managed stacks would be useful. For instance, my Dockge is launched by TrueNAS, and shows a disabled "ix-dockge" stack.
